jest.config.ts

```diff
@@ -96,9 +96,19 @@ function getTestsForGroup(
 
   // We sort files by path so that we try and improve the transformer cache hit rate.
   // Colocated domain specific files are likely to require other domain specific files.
-  const testsSortedByPath = Array.from(tests.entries()).sort((a, b) =>
-    a[0].localeCompare(b[0])
-  );
+  const testsSortedByPath = Array.from(tests.entries()).sort((a, b) => {
+    // WidgetBuilder tests are a special case as they can sometimes take a long time to run (3-5 minutes)
+    // As such, we want to ensure that they are ran in the same group as other widget builder tests.
+    // We do this by sorting them by the path of the widget builder test which ensures they are started by the first job
+    // in the CI group and that all of the tests actually run in the same group.
+    if (a[0].includes('widgetBuilder')) {
```

Comment:
hm, what if this starts to happen frequently for other tests as well?

Reply:
We can tackle it then :)

Reply:
On a more serious note, we do have monitoring for slow tests so if it does end up happening, we should investigate it then because it depends on a number of factors (are there many new slow tests, can we fix them etc)

Reply:
ok, it makes sense.I'm just concerned that this isn't a very scalable solution